packages/shared/styles/components/molecules/SfSelect.scss

```diff
@@ -28,7 +28,6 @@
   box-sizing: border-box;
   width: var(--select-width);
   height: var(--select-height);
-  padding: var(--select-padding, 0 0 calc(var(--font-xs) * 1.2) 0);
```

Comment:
for now, I think its better to keep this and change another padding variable name to '--select-selected-padding', wdyt ?

Reply:
ok I did that and in fact it resolved the problem in shipping mentioned below :)